src/EntityFramework/EntityFramework.csproj

```diff
@@ -7,6 +7,9 @@
     <Description>EntityFramework.dll</Description>
     <AssemblyVersion>6.0.0.0</AssemblyVersion>
     <DefineConstants>$(DefineConstants);INTERNALS_INVISIBLE</DefineConstants>
+
+    <!-- Properties\Resources.tt expects a path that does not match the class in Properties\Resources.cs -->
```

Comment:
Is it worth changing the code to use the conventional name?

Reply:
This is a great question I leave up to the maintainers. It didn't seem trivial, but I don't know the history of the `Resources.tt` approach or the details of its implementation.